sway-core/src/language/ty/module.rs

```diff
@@ -20,6 +20,9 @@ pub struct TyModule {
     pub attributes: transform::AttributesMap,
 }
 
+unsafe impl Send for TyModule {}
+unsafe impl Sync for TyModule {}
```

Comment:
I'm not really comfortable with declaring the whole AST safe to pass and modify accross threads. This at least requires a justification of why it's correct.